### DISCUSSION TITLE
Plot log files / created date getting messed up due to script to support Hpool

### DISCUSSION THREAD

**Discussion by Woogz-git** [guydavis/machinaris#138] [Q&A] [answered]
I'm running Machinaris in plotter mode in Unraid. Using MadMax. I have a separate windows VM to run the Hpool miner.

I was having a problem where the Hpool miner considered my plots invalid (due to a permissions issue I believe) so I came across a post that recommended scheduling the following user script:

chown nobody:users /mnt/user/XXX-YOU-SHARE-HERE-XXX/plot*.plot && chmod 755 /mnt/user/XXX-YOU-SHARE-HERE-XXX/plot*.plot

It works like a charm, shoutout to Bit by Bit Tutorials / Cryptomon.

only problem though is that it resets the created date for ALL of my plots to the last time it ran and it makes it so that I can't view the plot log file.  See below. Any fix for this? 

<img width="225" alt="Screen Shot 2021-07-04 at 7 19 29 AM" src="https://user-images.githubusercontent.com/21038356/124392332-dfb6d700-dcc2-11eb-9586-3869e8382622.png">
<img width="804" alt="Screen Shot 2021-07-04 at 7 19 14 AM" src="https://user-images.githubusercontent.com/21038356/124392333-dfb6d700-dcc2-11eb-825e-2d90bc3590c8.png">

**Comment by guydavis** [comment, marked as answer]
Hi!  Yes, let me see if I can find a solution to create vs mod time.  Check out the [latest test version](https://github.com/guydavis/machinaris/wiki/Releases) and see if the dates are improved.

The second error likely indicates the plot was created using some other plotting software, besides Machinaris.  In that case, the analysis of such plots is not available as there is not plot job file (created by Plotman in Machinaris) to analyze.

**Reply by Woogz-git** [reply]
I'll check out the latest test version, thanks!

All of those plots were created using MadMax plotter within Machinaris but will show that message if I've run the user script since the plot was created. The YouTuber where I got the script says that the most recent version of MadMax has fixed the mover to address this so that the script is no longer needed so we'll see.

**Reply by guydavis** [reply]
Great.  Machinaris (via Plotman) executes the Analyze feature by reviewing the plot job log created by Plotman inside Machinaris.  If these plots were truly made by Machinaris, can you find them in the Plotman logs folder?

```
docker exec -it machinaris bash
cd /root/.chia/plotman/logs
grep MY_PLOT_ID *.log
```
If you can replace MY_PLOT_ID with a plot_id from the Machinaris farming page and grep returns a hit, then we may have an error, in which case [check your logs](https://github.com/guydavis/machinaris/wiki/Troubleshooting).  However, no hit by grep means the plotting job log is not there anymore, perhaps deleted?
